src/lightstep-datasource/datasource.js

```diff
@@ -329,15 +345,23 @@ export class LightStepDatasource {
       const skip = Math.ceil(exemplars.length / maxDataPoints);
       exemplars = exemplars.filter((ignored, index) => index % skip === 0);
     }
+
+    const dp = exemplars.map(exemplar => {
+      return {
+        0: exemplar["duration_micros"] / 1000,
+        1: moment(((exemplar["oldest_micros"] + exemplar["youngest_micros"]) / 2) / 1000),
+        2: this.traceLink(exemplar),
+      };
+    });
     return [{
       target: name,
-      datapoints: exemplars.map(exemplar => {
-        return {
-          0: exemplar["duration_micros"] / 1000,
-          1: moment(((exemplar["oldest_micros"] + exemplar["youngest_micros"]) / 2) / 1000),
-          2: this.traceLink(exemplar),
-        };
-      }),
+      datapoints: dp,
+      meta: {
```

Comment:
grabbing the tracelinks and indexing by array index since this is how we're able to identify the exemplar that was clicked